plugins/git4idea/src/git4idea/annotate/GitFileAnnotation.java

```diff
@@ -265,12 +265,12 @@ protected void showAffectedPaths(int lineNum) {
       if (lineNum >= 0 && lineNum < myLines.size()) {
         LineInfo info = myLines.get(lineNum);
 
-        AbstractVcsHelperImpl.loadAndShowCommittedChangesDetails(myProject, info.getRevisionNumber(), myFilePath, () -> getRevisionsChangesProvider().getChangesIn(lineNum));
+        AbstractVcsHelperImpl.loadAndShowCommittedChangesDetails(myProject, info.getCommitInfo().getRevisionNumber(), myFilePath, () -> getRevisionsChangesProvider().getChangesIn(lineNum));
       }
     }
   }
 
-  public static class LineInfo {
+  public static class CommitInfo {
```

Comment:
Is there are a reason, why `CommitInfo` should be public?
Its methods are duplicated by `LineInfo`.